support-frontend/assets/pages/digital-subscriber-thank-you/digitalSubscriptionThankYou.tsx

```diff
@@ -54,11 +52,7 @@ export function DigitalSubscriptionThankYou(): JSX.Element {
 	): ThankYouModuleType[] => (condtion ? [moduleType] : []);
 
 	const thankYouModules: ThankYouModuleType[] = [
-		...maybeThankYouModule(isNewAccount, 'signUp'),
```

Comment:
Is this still used?

Reply:
The page? Yes, but we've agreed it's low priority so we can do without this module